include/ofi_atomic_queue.h

```diff
@@ -135,9 +135,10 @@ static inline void name ## _init(struct name *aq, size_t size)	\
 static inline struct name * name ## _create(size_t size)	\
 {								\
 	struct name *aq;					\
-	aq = (struct name*) calloc(1, sizeof(*aq) +		\
-		sizeof(struct name ## _entry) *			\
-		(roundup_power_of_two(size)));			\
+	aq = (struct name *) aligned_alloc(			\
+			OFI_CACHE_LINE_SIZE, sizeof(*aq) +	\
+			sizeof(struct name ## _entry) *		\
+			(roundup_power_of_two(size)));		\
```

Comment:
Now the allocation is uninitialized.  The `init` macro would initialize most of the fields but leave `entry.noop` and `entry.buf` uninitialized. Would that be an issue? especially the `noop` field.

Reply:
The buf doesn't need to be initialized but you're right, the noop should get initialized to false. Good catch, thanks!